setup.py

```diff
@@ -203,6 +203,18 @@ def build_extension(self, ext):
         # build project
         subprocess.check_call(['cmake', '-S.', '-B', build_temp] + cmake_args)
         subprocess.check_call(['cmake', '--build', build_temp] + build_args)
```

Comment:
Is it possible to merge L199-202 with L203-205?

Reply:
@GACLove